src/averagePerceptronTagger.jl

```diff
@@ -0,0 +1,326 @@
+using DataStructures
+using Random
+using BSON
+
+"""AVERAGE PERCEPTRON MODEL"""
+
+"""
+This module contains the Average Perceptron model
+which was original implemented by Matthew Honnibal.
+
+The model learns by basic perceptron algorithm
+but after all iterations weights are being averaged
+"""
+mutable struct AveragePerceptron
+    classes :: Set
+    weights :: Dict
+    _totals :: DefaultDict
+    _tstamps :: DefaultDict

```

Comment:
We can specify a default value to the `DefaultDict` ,  if there a key whose value is yet defined in the dict then if we reference it in `Dict` it will show and error but if we do same with `DefaultDict` the default value will be assigned to that key and we can do any operation even if didn't specify any value for that key the dict earlier.
Ex.
a = DefaultDict(0)
a["some_key"] += 5
this won't show any error because by default 0 is assigned to "some_key" in the dict.

b = Dict{String, Int64}()
b["some_key"] += 5
this will show error because "some_key " was not defined earlier.